plugins/grpc-transport-auth/src/test/java/com/github/bsideup/liiklus/transport/grpc/GRPCTLSTest.java

```diff
@@ -0,0 +1,71 @@
+package com.github.bsideup.liiklus.transport.grpc;
+
+import com.github.bsideup.liiklus.ApplicationRunner;
+import com.github.bsideup.liiklus.GRPCLiiklusClient;
+import com.github.bsideup.liiklus.protocol.PublishRequest;
+import com.google.protobuf.ByteString;
+import io.grpc.Server;
+import io.grpc.StatusRuntimeException;
+import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.NettyChannelBuilder;
+import lombok.SneakyThrows;
+import org.junit.Test;
+import org.pf4j.PluginManager;
+import org.springframework.context.ApplicationContext;
+import org.springframework.util.ResourceUtils;
+
+import javax.net.ssl.SSLException;
+import java.io.FileNotFoundException;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class GRPCTLSTest {
+
+    @SneakyThrows
+    static int getGRPCPort(ApplicationContext ctx) {
```

Comment:
duplicate of https://github.com/bsideup/liiklus/blob/fae66dc262cbef61f79fef9343e9ca9341c40fba/plugins/grpc-transport-auth/src/test/java/com/github/bsideup/liiklus/transport/grpc/GRPCAuthTest.java#L50

Reply:
yep, removed. Copypasta artifact for debug